packages/common/src/utils/faker/fakeCertificateData.ts

```diff
@@ -255,6 +255,57 @@ export const fakeMedicalInvestigationListElement = fakeDataElement<ConfigUeMedic
   }
 )
 
+export const fakeEmptyMedicalInvestigationListElement = fakeDataElement<ConfigUeMedicalInvestigationList, ValueMedicalInvestigationList>(
```

Comment:
How does this differ from `fakeMedicalInvestigationListElement` ?

Reply:
looks like it's using adding an empty value list, so you could replace this with `fakeMedicalInvestigationListElement({value: { list: [] }})`

Reply:
Good suggestion! Fixed it now.